script/pip_check

```diff
@@ -3,7 +3,7 @@ PIP_CACHE=$1
 
 # Number of existing dependency conflicts
 # Update if a PR resolves one!
-DEPENDENCY_CONFLICTS=3
+DEPENDENCY_CONFLICTS=4
```

Comment:
This is coming right back out https://github.com/home-assistant/core/pull/90991 but needed to leave it in to get a clean CI run